static/scripts/teams.js

```diff
@@ -196,6 +197,25 @@ $(document).ready(() => {
 			payload: { userId },
 		});
 
+		const $deleteInfoAlert = $leaveTeamModal.find('.alert');
+		const $fileList = $leaveTeamModal.find('.file-list');
+
+		if (files && files.length > 0) {
+			let fileListHtmlLi = '';
```

Comment:
U can simplify with const fileListHtml = files.map(file => `<li>${file}</li>`).join(''); if it is array

Reply:
Changed according to suggestion